streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/transfer/TransferHandler.java

```diff
@@ -10,12 +11,8 @@
 public class TransferHandler {
 
   static {
-    try {
-      Class.forName(RayNativeRuntime.class.getName());
-    } catch (ClassNotFoundException e) {
-      throw new RuntimeException(e);
-    }
-    JniUtils.loadLibrary("streaming_java");
+    JniUtils.loadLibrary(null, BinaryFileUtil.CORE_WORKER_JAVA_LIBRARY, true);
```

Comment:
Provide an overloaded `loadLibrary` method which receives library name instead of pass null